### PR TITLE
Feat(get-funding-channels): Add documentation for list funding channels endpoint

### PR DESCRIPTION
**Pull request by chanjant**
document list api in docs.immersve.com

Ticket Link: https://www.notion.so/immersve/document-get-and-list-apis-in-docs-immersve-com-6bb51b01141a4581ab2749f92b80981d

Screenshot 2023-12-11 at 9 46 47 am

Screenshot 2023-12-11 at 9 28 41 am

### CODE REVIEW

File: openapi/endpoints/funding-channel/funding-channels-get.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+get:
+  tags:
+    - Funding Channel
+  summary: List funding channels
+  description: Retrieves a list of fundingChannels for the given account
```
Comment (tonykha123) [inline openapi/endpoints/funding-channel/funding-channels-get.yaml:5]:
```suggestion
  description: Retrieves a list of Funding Channels for the given account
```

Can we do this everywhere

File: openapi/endpoints/funding-channel/models/get-funding-channel.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+type: object
+properties:
+  id:
+    type: string
+    description: Funding channel primary identifier
+    example: fundingChannelId1
```
Comment (tonykha123) [inline openapi/endpoints/funding-channel/models/get-funding-channel.yaml:6]:
```suggestion
    example: 315bad4e81ce0f26966a41b9d451638b
```

actual id use on the other endpoints would be good

File: openapi/endpoints/funding-channel/models/get-funding-channel.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    type: string
+    description: Timestamp of funding channel creation
+    example: "2023-08-14T12:32:54"
+  address:
+    type: string
+    description: Fund storage smart contract address
+    example: "0x2e429C4324227F70f0101a17214d249cb04035Ec0"
+  fundingType:
+    type: string
+    description: The type of funding channel registered
```
Comment (tonykha123) [inline openapi/endpoints/funding-channel/models/get-funding-channel.yaml:17]:
```suggestion
    description: The type of funding channel registered.
```
We should use sentences

**Review by tonykha123 — COMMENTED** [review]
Looks good overall. some feedback to address, you can refer to the other endpoint docs as an example

**Review by ncjones — APPROVED** [review]

**Pull request merged**